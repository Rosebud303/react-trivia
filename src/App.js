import React, { Component } from 'react';

import TriviaList from './TriviaList.js';
import Controls from './Controls.js'

export default class App extends Component {
  constructor () {
    super();

    this.state = {
      triviaQuestions: [],
      selectedLimit: 5
    }

    this.setLimit = this.setLimit.bind(this);
  }

  componentDidMount() {
    fetch('https://opentdb.com/api.php?amount=10')
      .then(response => response.json())
      .then(questions => {
        this.setState({
          triviaQuestions: questions.results
        })
      })
      .catch(error => {
        throw new Error(error);
      });
  }

  setLimit(limit) {
    this.setState({
      selectedLimit: limit
    })
  }

  render () {
    return (
      <div>
        <h1>Trivia!</h1>
        <Controls setLimit={this.setLimit} />
        <TriviaList 
          triviaQuestions={this.state.triviaQuestions}
          questionCount={this.state.selectedLimit} />
      </div>
    )
  }
}