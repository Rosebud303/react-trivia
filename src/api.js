const categories = [{"id":22,"name":"Geography"},{"id":9,"name":"General Knowledge"},{"id":23,"name":"History"}];
const triviaQuestions = [{category:"Entertainment: Video Games",type:"multiple",difficulty:"hard",question:"When was the first Call of Duty question released?",correct_answer:"October 29, 2003",incorrect_answers:["December 1, 2003","November 14, 2002","July 18, 2004"],id:1},{category:"Entertainment: Books",type:"multiple",difficulty:"medium",question:"How many books are in the Chronicles of Narnia series?",correct_answer:"7",incorrect_answers:["6","8","5"],id:2},{category:"Entertainment: Film",type:"boolean",difficulty:"easy",question:"Han Solo's co-pilot and best friend, 'Chewbacca', is an Ewok.",correct_answer:"False",incorrect_answers:["True"],id:3},{category:"General Knowledge",type:"multiple",difficulty:"medium",question:"The Tsar Bomba, the most powerful nuclear bomb ever tested, had a yield of 50 megatons but theoretically had a maximum yield of how much?",correct_answer:"100 Megatons",incorrect_answers:["200 Megatons","75 Megatons","150 Megatons"],id:4},{category:"Entertainment: Video Games",type:"multiple",difficulty:"medium",question:"Which of these is NOT a terrorist faction in Counter-Strike (2000)?",correct_answer:"Midwest Militia",incorrect_answers:["Phoenix Connection","Elite Crew","Guerrilla Warfare"],id:5},{category:"Entertainment: Video Games",type:"multiple",difficulty:"medium",question:"Which of these games was the earliest known first-person shooter with a known time of publication?",correct_answer:"Spasim",incorrect_answers:["Doom","Wolfenstein","Quake"],id:6},{category:"Entertainment: Film",type:"multiple",difficulty:"hard",question:"What did Alfred Hitchcock use as blood in the film 'Psycho'? ",correct_answer:"Chocolate syrup",incorrect_answers:["Ketchup","Red food coloring","Maple syrup"],id:7},{category:"Entertainment: Video Games",type:"multiple",difficulty:"medium",question:"How many Star Spirits do you rescue in the Nintendo 64 video game 'Paper Mario'?",correct_answer:"7",incorrect_answers:["5","10","12"],id:8},{category:"Entertainment: Film",type:"multiple",difficulty:"medium",question:"Who played Batman in the 1997 film 'Batman and Robin'?",correct_answer:"George Clooney",incorrect_answers:["Michael Keaton","Val Kilmer","Christian Bale"],id:9},{category:"Entertainment: Film",type:"multiple",difficulty:"medium",question:"This movie contains the quote, 'What we've got here is a failure to communicate.'",correct_answer:"Cool Hand Luke",incorrect_answers:["Bonnie and Clyde","The Graduate","In the Heat of the Night"],id:10},{category:"Entertainment: Books",type:"multiple",difficulty:"medium",question:"Who wrote the children's story 'The Little Match Girl'?",correct_answer:"Hans Christian Andersen",incorrect_answers:["Charles Dickens","Lewis Carroll","Oscar Wilde"],id:11},{category:"Science & Nature",type:"multiple",difficulty:"hard",question:"An organic compound is considered an alcohol if it has what functional group?",correct_answer:"Hydroxyl",incorrect_answers:["Carbonyl","Alkyl","Aldehyde"],id:12},{category:"Mythology",type:"multiple",difficulty:"medium",question:"Neptune's greek name was...",correct_answer:"Poseidon",incorrect_answers:["Ares","Zeus","Apollo"],id:13},{category:"General Knowledge",type:"multiple",difficulty:"medium",question:"What is a 'dakimakura'?",correct_answer:"A body pillow",incorrect_answers:["A Chinese meal, essentially composed of fish","A yoga posture","A word used to describe two people who truly love each other"],id:14},{category:"Geography",type:"multiple",difficulty:"medium",question:"Which of these countries is NOT located in Africa?",correct_answer:"Suriname",incorrect_answers:["Burkina Faso","Mozambique","Algeria"],id:15},{category:"Entertainment: Comics",type:"multiple",difficulty:"medium",question:"What is the name of the comic about a young boy, and a tiger who is actually a stuffed animal?",correct_answer:"Calvin and Hobbes",incorrect_answers:["Winnie the Pooh","Albert and Pogo","Peanuts"],id:16},{category:"History",type:"boolean",difficulty:"easy",question:"The United States was a member of the League of Nations.",correct_answer:"False",incorrect_answers:["True"],id:17},{category:"Entertainment: Video Games",type:"multiple",difficulty:"medium",question:"In the 2002 video game 'Kingdom Hearts', how many playable worlds were there?",correct_answer:"14",incorrect_answers:["13","16","11"],id:18},{category:"General Knowledge",type:"multiple",difficulty:"hard",question:"What is the romanized Korean word for 'heart'?",correct_answer:"Simjang",incorrect_answers:["Aejeong","Jeongsin","Segseu"],id:19},{category:"Sports",type:"multiple",difficulty:"hard",question:"Where was the Games of the XXII Olympiad held?",correct_answer:"Moscow",incorrect_answers:["Barcelona","Tokyo","Los Angeles"],id:20},{category:"History",type:"multiple",difficulty:"hard",question:"When was the SS or Schutzstaffel established?",correct_answer:"April 4th, 1925",incorrect_answers:["September 1st, 1941","March 8th, 1935","February 21st, 1926"],id:21},{category:"General Knowledge",type:"boolean",difficulty:"easy",question:"You can legally drink alcohol while driving in Mississippi.",correct_answer:"True",incorrect_answers:["False"],id:22},{category:"Entertainment: Video Games",type:"multiple",difficulty:"medium",question:"Which of these is the name of a cut enemy from 'Half-Life 2'?",correct_answer:"Hydra",incorrect_answers:["Cthulu","Spike","Tremor"],id:23},{category:"Entertainment: Film",type:"multiple",difficulty:"medium",question:"Who played Baron Victor Frankestein in the 1957 Hammer horror film 'The Curse of Frankenstein'?",correct_answer:"Peter Cushing",incorrect_answers:["Boris Karloff","Vincent Price","Lon Chaney Jr."],id:24},{category:"Vehicles",type:"multiple",difficulty:"medium",question:"Which car brand does NOT belong to General Motors?",correct_answer:"Ford",incorrect_answers:["Buick","Cadillac","Chevrolet"],id:25},{category:"Vehicles",type:"multiple",difficulty:"medium",question:"What do the 4 Rings in Audi's Logo represent?",correct_answer:"Previously independent automobile manufacturers",incorrect_answers:["States in which Audi makes the most sales","Main cities vital to Audi","Countries in which Audi makes the most sales"],id:26},{category:"Animals",type:"multiple",difficulty:"hard",question:"What is the Gray Wolf's scientific name?",correct_answer:"Canis Lupus",incorrect_answers:["Canis Aureus","Canis Latrans","Canis Lupus Lycaon"],id:27},{category:"Entertainment: Video Games",type:"multiple",difficulty:"easy",question:"In vanilla Minecraft, which of the following cannot be made into a block?",correct_answer:"Charcoal",incorrect_answers:["Coal","Wheat","String"],id:28},{category:"Entertainment: Video Games",type:"multiple",difficulty:"medium",question:"Which is not a playable character in the 2005 video game Killer7?",correct_answer:"Frank Smith",incorrect_answers:["Mask de Smith","Dan Smith","Coyote Smith"],id:29},{category:"Entertainment: Television",type:"multiple",difficulty:"easy",question:"What country is Cory in the House set in?",correct_answer:"The United States of America",incorrect_answers:["Canada","Venezuala","England"],id:30},{category:"History",type:"multiple",difficulty:"easy",question:"What was the first sport to have been played on the moon?",correct_answer:"Golf",incorrect_answers:["Football","Tennis","Soccer"],id:31},{category:"Vehicles",type:"multiple",difficulty:"hard",question:"What engine is in the Lexus SC400?",correct_answer:"1UZ-FE",incorrect_answers:["2JZ-GTE","7M-GTE","5M-GE"],id:32},{category:"Entertainment: Music",type:"multiple",difficulty:"hard",question:"Which year was the album 'Floral Shoppe' by Macintosh Plus released?",correct_answer:"2011",incorrect_answers:["2014","2013","2012"],id:33},{category:"Entertainment: Television",type:"multiple",difficulty:"hard",question:"Which country does the YouTuber 'SinowBeats' originate from?",correct_answer:"Scotland",incorrect_answers:["England","Sweden","Germany"],id:34},{category:"Entertainment: Film",type:"boolean",difficulty:"hard",question:"'Cube', 'Cube 2: Hypercube' and 'Cube Zero' were directed by the same person.",correct_answer:"False",incorrect_answers:["True"],id:35},{category:"Entertainment: Music",type:"boolean",difficulty:"medium",question:"EDM label Monstercat signs tracks instead of artists.",correct_answer:"True",incorrect_answers:["False"],id:36},{category:"Geography",type:"multiple",difficulty:"medium",question:"How tall is One World Trade Center in New York City?",correct_answer:"1,776 ft",incorrect_answers:["1,888 ft","1,225 ft","1,960 ft"],id:37},{category:"Geography",type:"multiple",difficulty:"hard",question:"What is the name of rocky region that spans most of eastern Canada?",correct_answer:"Canadian Shield",incorrect_answers:["Rocky Mountains","Appalachian Mountains","Himalayas"],id:38},{category:"History",type:"multiple",difficulty:"easy",question:"How was Socrates executed?",correct_answer:"Poison",incorrect_answers:["Decapitation","Firing squad","Crucifixion "],id:39},{category:"Sports",type:"multiple",difficulty:"easy",question:"What year did the New Orleans Saints win the Super Bowl?",correct_answer:"2009",incorrect_answers:["2008","2010","2011"],id:40},{category:"General Knowledge",type:"multiple",difficulty:"medium",question:"When was Hubba Bubba first introduced?",correct_answer:"1979",incorrect_answers:["1984","1972","1980"],id:41},{category:"Geography",type:"multiple",difficulty:"easy",question:"What is the capital of Spain?",correct_answer:"Madrid",incorrect_answers:["Barcelona","Sevilla","Toledo"],id:42},{category:"Entertainment: Cartoon & Animations",type:"multiple",difficulty:"easy",question:"In the show 'Steven Universe', who are the main two employees of The Big Donut?",correct_answer:"Sadie and Lars",incorrect_answers:["Steven and James","Erik and Julie","Bob and May"],id:43},{category:"Geography",type:"multiple",difficulty:"easy",question:"What is the capital of India?",correct_answer:"New Delhi",incorrect_answers:["Bejing","Montreal","Tithi"],id:44},{category:"Entertainment: Television",type:"multiple",difficulty:"easy",question:"In season one of the Netflix political drama 'House of Cards', what government position does Frank Underwood hold?",correct_answer:"House Majority Whip",incorrect_answers:["Attorney General","President","Chief of Staff"],id:45},{category:"History",type:"multiple",difficulty:"medium",question:"What was the name of one of the surviving palaces of Henry VIII located near Richmond, London?",correct_answer:"Hampton Court",incorrect_answers:["St James's Palace","Buckingham Palace","Coughton Court"],id:46},{category:"Entertainment: Japanese Anime & Manga",type:"multiple",difficulty:"hard",question:"Which one of these characters is from 'Legendz : Tale of the Dragon Kings'?",correct_answer:"Shiron",incorrect_answers:["Jack","Axia","Drum"],id:47},{category:"Entertainment: Music",type:"multiple",difficulty:"medium",question:"What was David Bowie's real surname?",correct_answer:"Jones",incorrect_answers:["Johnson","Edwards","Carter"],id:48},{category:"Entertainment: Japanese Anime & Manga",type:"multiple",difficulty:"medium",question:"In Pok&eacute;mon Chronicles, why was Misty afraid of Gyarados?",correct_answer:"She crawled into it's mouth as a baby.",incorrect_answers:["She found it scary.","She was badly injured from it.","It is part Bug."],id:49},{category:"Celebrities",type:"multiple",difficulty:"medium",title:"In which of these TV shows did the chef Gordon Ramsay not appear?",correct_answer:"Auction Hunters",incorrect_answers:["Ramsay's Kitchen Nightmares","Hotel Hell","Hell's Kitchen"],id:50}];

export {
  categories,
  triviaQuestions
}