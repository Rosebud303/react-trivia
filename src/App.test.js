import React from 'react';
import App from './App';
import { triviaQuestions, categories } from './api.js';
import { shallow } from 'enzyme';

describe('App', () => {

  let wrapper;

  beforeEach(() => {
    wrapper = shallow(
      <App />
    );
  });

  
  it('should match the snapshot with all data passed in', () => {
    expect(wrapper).toMatchSnapshot();
  });


  it('should have the proper default state', () => {
    expect(wrapper.state()).toEqual({
      triviaQuestions: triviaQuestions,
      selectedLimit: 5
    })
  });

  it('should update the limit when setLimit is called', () => {
    expect(wrapper.state('selectedLimit')).toEqual(5);
    wrapper.instance().setLimit(7);
    expect(wrapper.state('selectedLimit')).toEqual(7);
  });

});
