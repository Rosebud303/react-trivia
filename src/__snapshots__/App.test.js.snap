// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should match the snapshot with all data passed in 1`] = `
<div>
  <h1>
    Trivia!
  </h1>
  <Controls
    setLimit={[Function]}
  />
  <TriviaList
    questionCount={5}
    triviaQuestions={
      Array [
        Object {
          "category": "Entertainment: Video Games",
          "correct_answer": "October 29, 2003",
          "difficulty": "hard",
          "id": 1,
          "incorrect_answers": Array [
            "December 1, 2003",
            "November 14, 2002",
            "July 18, 2004",
          ],
          "title": "When was the first Call of Duty title released?",
          "type": "multiple",
        },
        Object {
          "category": "Entertainment: Books",
          "correct_answer": "7",
          "difficulty": "medium",
          "id": 2,
          "incorrect_answers": Array [
            "6",
            "8",
            "5",
          ],
          "title": "How many books are in the Chronicles of Narnia series?",
          "type": "multiple",
        },
        Object {
          "category": "Entertainment: Film",
          "correct_answer": "False",
          "difficulty": "easy",
          "id": 3,
          "incorrect_answers": Array [
            "True",
          ],
          "title": "Han Solo's co-pilot and best friend, 'Chewbacca', is an Ewok.",
          "type": "boolean",
        },
        Object {
          "category": "General Knowledge",
          "correct_answer": "100 Megatons",
          "difficulty": "medium",
          "id": 4,
          "incorrect_answers": Array [
            "200 Megatons",
            "75 Megatons",
            "150 Megatons",
          ],
          "title": "The Tsar Bomba, the most powerful nuclear bomb ever tested, had a yield of 50 megatons but theoretically had a maximum yield of how much?",
          "type": "multiple",
        },
        Object {
          "category": "Entertainment: Video Games",
          "correct_answer": "Midwest Militia",
          "difficulty": "medium",
          "id": 5,
          "incorrect_answers": Array [
            "Phoenix Connection",
            "Elite Crew",
            "Guerrilla Warfare",
          ],
          "title": "Which of these is NOT a terrorist faction in Counter-Strike (2000)?",
          "type": "multiple",
        },
        Object {
          "category": "Entertainment: Video Games",
          "correct_answer": "Spasim",
          "difficulty": "medium",
          "id": 6,
          "incorrect_answers": Array [
            "Doom",
            "Wolfenstein",
            "Quake",
          ],
          "title": "Which of these games was the earliest known first-person shooter with a known time of publication?",
          "type": "multiple",
        },
        Object {
          "category": "Entertainment: Film",
          "correct_answer": "Chocolate syrup",
          "difficulty": "hard",
          "id": 7,
          "incorrect_answers": Array [
            "Ketchup",
            "Red food coloring",
            "Maple syrup",
          ],
          "title": "What did Alfred Hitchcock use as blood in the film 'Psycho'? ",
          "type": "multiple",
        },
        Object {
          "category": "Entertainment: Video Games",
          "correct_answer": "7",
          "difficulty": "medium",
          "id": 8,
          "incorrect_answers": Array [
            "5",
            "10",
            "12",
          ],
          "title": "How many Star Spirits do you rescue in the Nintendo 64 video game 'Paper Mario'?",
          "type": "multiple",
        },
        Object {
          "category": "Entertainment: Film",
          "correct_answer": "George Clooney",
          "difficulty": "medium",
          "id": 9,
          "incorrect_answers": Array [
            "Michael Keaton",
            "Val Kilmer",
            "Christian Bale",
          ],
          "title": "Who played Batman in the 1997 film 'Batman and Robin'?",
          "type": "multiple",
        },
        Object {
          "category": "Entertainment: Film",
          "correct_answer": "Cool Hand Luke",
          "difficulty": "medium",
          "id": 10,
          "incorrect_answers": Array [
            "Bonnie and Clyde",
            "The Graduate",
            "In the Heat of the Night",
          ],
          "title": "This movie contains the quote, 'What we've got here is a failure to communicate.'",
          "type": "multiple",
        },
        Object {
          "category": "Entertainment: Books",
          "correct_answer": "Hans Christian Andersen",
          "difficulty": "medium",
          "id": 11,
          "incorrect_answers": Array [
            "Charles Dickens",
            "Lewis Carroll",
            "Oscar Wilde",
          ],
          "title": "Who wrote the children's story 'The Little Match Girl'?",
          "type": "multiple",
        },
        Object {
          "category": "Science & Nature",
          "correct_answer": "Hydroxyl",
          "difficulty": "hard",
          "id": 12,
          "incorrect_answers": Array [
            "Carbonyl",
            "Alkyl",
            "Aldehyde",
          ],
          "title": "An organic compound is considered an alcohol if it has what functional group?",
          "type": "multiple",
        },
        Object {
          "category": "Mythology",
          "correct_answer": "Poseidon",
          "difficulty": "medium",
          "id": 13,
          "incorrect_answers": Array [
            "Ares",
            "Zeus",
            "Apollo",
          ],
          "title": "Neptune's greek name was...",
          "type": "multiple",
        },
        Object {
          "category": "General Knowledge",
          "correct_answer": "A body pillow",
          "difficulty": "medium",
          "id": 14,
          "incorrect_answers": Array [
            "A Chinese meal, essentially composed of fish",
            "A yoga posture",
            "A word used to describe two people who truly love each other",
          ],
          "title": "What is a 'dakimakura'?",
          "type": "multiple",
        },
        Object {
          "category": "Geography",
          "correct_answer": "Suriname",
          "difficulty": "medium",
          "id": 15,
          "incorrect_answers": Array [
            "Burkina Faso",
            "Mozambique",
            "Algeria",
          ],
          "title": "Which of these countries is NOT located in Africa?",
          "type": "multiple",
        },
        Object {
          "category": "Entertainment: Comics",
          "correct_answer": "Calvin and Hobbes",
          "difficulty": "medium",
          "id": 16,
          "incorrect_answers": Array [
            "Winnie the Pooh",
            "Albert and Pogo",
            "Peanuts",
          ],
          "title": "What is the name of the comic about a young boy, and a tiger who is actually a stuffed animal?",
          "type": "multiple",
        },
        Object {
          "category": "History",
          "correct_answer": "False",
          "difficulty": "easy",
          "id": 17,
          "incorrect_answers": Array [
            "True",
          ],
          "title": "The United States was a member of the League of Nations.",
          "type": "boolean",
        },
        Object {
          "category": "Entertainment: Video Games",
          "correct_answer": "14",
          "difficulty": "medium",
          "id": 18,
          "incorrect_answers": Array [
            "13",
            "16",
            "11",
          ],
          "title": "In the 2002 video game 'Kingdom Hearts', how many playable worlds were there?",
          "type": "multiple",
        },
        Object {
          "category": "General Knowledge",
          "correct_answer": "Simjang",
          "difficulty": "hard",
          "id": 19,
          "incorrect_answers": Array [
            "Aejeong",
            "Jeongsin",
            "Segseu",
          ],
          "title": "What is the romanized Korean word for 'heart'?",
          "type": "multiple",
        },
        Object {
          "category": "Sports",
          "correct_answer": "Moscow",
          "difficulty": "hard",
          "id": 20,
          "incorrect_answers": Array [
            "Barcelona",
            "Tokyo",
            "Los Angeles",
          ],
          "title": "Where was the Games of the XXII Olympiad held?",
          "type": "multiple",
        },
        Object {
          "category": "History",
          "correct_answer": "April 4th, 1925",
          "difficulty": "hard",
          "id": 21,
          "incorrect_answers": Array [
            "September 1st, 1941",
            "March 8th, 1935",
            "February 21st, 1926",
          ],
          "title": "When was the SS or Schutzstaffel established?",
          "type": "multiple",
        },
        Object {
          "category": "General Knowledge",
          "correct_answer": "True",
          "difficulty": "easy",
          "id": 22,
          "incorrect_answers": Array [
            "False",
          ],
          "title": "You can legally drink alcohol while driving in Mississippi.",
          "type": "boolean",
        },
        Object {
          "category": "Entertainment: Video Games",
          "correct_answer": "Hydra",
          "difficulty": "medium",
          "id": 23,
          "incorrect_answers": Array [
            "Cthulu",
            "Spike",
            "Tremor",
          ],
          "title": "Which of these is the name of a cut enemy from 'Half-Life 2'?",
          "type": "multiple",
        },
        Object {
          "category": "Entertainment: Film",
          "correct_answer": "Peter Cushing",
          "difficulty": "medium",
          "id": 24,
          "incorrect_answers": Array [
            "Boris Karloff",
            "Vincent Price",
            "Lon Chaney Jr.",
          ],
          "title": "Who played Baron Victor Frankestein in the 1957 Hammer horror film 'The Curse of Frankenstein'?",
          "type": "multiple",
        },
        Object {
          "category": "Vehicles",
          "correct_answer": "Ford",
          "difficulty": "medium",
          "id": 25,
          "incorrect_answers": Array [
            "Buick",
            "Cadillac",
            "Chevrolet",
          ],
          "title": "Which car brand does NOT belong to General Motors?",
          "type": "multiple",
        },
        Object {
          "category": "Vehicles",
          "correct_answer": "Previously independent automobile manufacturers",
          "difficulty": "medium",
          "id": 26,
          "incorrect_answers": Array [
            "States in which Audi makes the most sales",
            "Main cities vital to Audi",
            "Countries in which Audi makes the most sales",
          ],
          "title": "What do the 4 Rings in Audi's Logo represent?",
          "type": "multiple",
        },
        Object {
          "category": "Animals",
          "correct_answer": "Canis Lupus",
          "difficulty": "hard",
          "id": 27,
          "incorrect_answers": Array [
            "Canis Aureus",
            "Canis Latrans",
            "Canis Lupus Lycaon",
          ],
          "title": "What is the Gray Wolf's scientific name?",
          "type": "multiple",
        },
        Object {
          "category": "Entertainment: Video Games",
          "correct_answer": "Charcoal",
          "difficulty": "easy",
          "id": 28,
          "incorrect_answers": Array [
            "Coal",
            "Wheat",
            "String",
          ],
          "title": "In vanilla Minecraft, which of the following cannot be made into a block?",
          "type": "multiple",
        },
        Object {
          "category": "Entertainment: Video Games",
          "correct_answer": "Frank Smith",
          "difficulty": "medium",
          "id": 29,
          "incorrect_answers": Array [
            "Mask de Smith",
            "Dan Smith",
            "Coyote Smith",
          ],
          "title": "Which is not a playable character in the 2005 video game Killer7?",
          "type": "multiple",
        },
        Object {
          "category": "Entertainment: Television",
          "correct_answer": "The United States of America",
          "difficulty": "easy",
          "id": 30,
          "incorrect_answers": Array [
            "Canada",
            "Venezuala",
            "England",
          ],
          "title": "What country is Cory in the House set in?",
          "type": "multiple",
        },
        Object {
          "category": "History",
          "correct_answer": "Golf",
          "difficulty": "easy",
          "id": 31,
          "incorrect_answers": Array [
            "Football",
            "Tennis",
            "Soccer",
          ],
          "title": "What was the first sport to have been played on the moon?",
          "type": "multiple",
        },
        Object {
          "category": "Vehicles",
          "correct_answer": "1UZ-FE",
          "difficulty": "hard",
          "id": 32,
          "incorrect_answers": Array [
            "2JZ-GTE",
            "7M-GTE",
            "5M-GE",
          ],
          "title": "What engine is in the Lexus SC400?",
          "type": "multiple",
        },
        Object {
          "category": "Entertainment: Music",
          "correct_answer": "2011",
          "difficulty": "hard",
          "id": 33,
          "incorrect_answers": Array [
            "2014",
            "2013",
            "2012",
          ],
          "title": "Which year was the album 'Floral Shoppe' by Macintosh Plus released?",
          "type": "multiple",
        },
        Object {
          "category": "Entertainment: Television",
          "correct_answer": "Scotland",
          "difficulty": "hard",
          "id": 34,
          "incorrect_answers": Array [
            "England",
            "Sweden",
            "Germany",
          ],
          "title": "Which country does the YouTuber 'SinowBeats' originate from?",
          "type": "multiple",
        },
        Object {
          "category": "Entertainment: Film",
          "correct_answer": "False",
          "difficulty": "hard",
          "id": 35,
          "incorrect_answers": Array [
            "True",
          ],
          "title": "'Cube', 'Cube 2: Hypercube' and 'Cube Zero' were directed by the same person.",
          "type": "boolean",
        },
        Object {
          "category": "Entertainment: Music",
          "correct_answer": "True",
          "difficulty": "medium",
          "id": 36,
          "incorrect_answers": Array [
            "False",
          ],
          "title": "EDM label Monstercat signs tracks instead of artists.",
          "type": "boolean",
        },
        Object {
          "category": "Geography",
          "correct_answer": "1,776 ft",
          "difficulty": "medium",
          "id": 37,
          "incorrect_answers": Array [
            "1,888 ft",
            "1,225 ft",
            "1,960 ft",
          ],
          "title": "How tall is One World Trade Center in New York City?",
          "type": "multiple",
        },
        Object {
          "category": "Geography",
          "correct_answer": "Canadian Shield",
          "difficulty": "hard",
          "id": 38,
          "incorrect_answers": Array [
            "Rocky Mountains",
            "Appalachian Mountains",
            "Himalayas",
          ],
          "title": "What is the name of rocky region that spans most of eastern Canada?",
          "type": "multiple",
        },
        Object {
          "category": "History",
          "correct_answer": "Poison",
          "difficulty": "easy",
          "id": 39,
          "incorrect_answers": Array [
            "Decapitation",
            "Firing squad",
            "Crucifixion ",
          ],
          "title": "How was Socrates executed?",
          "type": "multiple",
        },
        Object {
          "category": "Sports",
          "correct_answer": "2009",
          "difficulty": "easy",
          "id": 40,
          "incorrect_answers": Array [
            "2008",
            "2010",
            "2011",
          ],
          "title": "What year did the New Orleans Saints win the Super Bowl?",
          "type": "multiple",
        },
        Object {
          "category": "General Knowledge",
          "correct_answer": "1979",
          "difficulty": "medium",
          "id": 41,
          "incorrect_answers": Array [
            "1984",
            "1972",
            "1980",
          ],
          "title": "When was Hubba Bubba first introduced?",
          "type": "multiple",
        },
        Object {
          "category": "Geography",
          "correct_answer": "Madrid",
          "difficulty": "easy",
          "id": 42,
          "incorrect_answers": Array [
            "Barcelona",
            "Sevilla",
            "Toledo",
          ],
          "title": "What is the capital of Spain?",
          "type": "multiple",
        },
        Object {
          "category": "Entertainment: Cartoon & Animations",
          "correct_answer": "Sadie and Lars",
          "difficulty": "easy",
          "id": 43,
          "incorrect_answers": Array [
            "Steven and James",
            "Erik and Julie",
            "Bob and May",
          ],
          "title": "In the show 'Steven Universe', who are the main two employees of The Big Donut?",
          "type": "multiple",
        },
        Object {
          "category": "Geography",
          "correct_answer": "New Delhi",
          "difficulty": "easy",
          "id": 44,
          "incorrect_answers": Array [
            "Bejing",
            "Montreal",
            "Tithi",
          ],
          "title": "What is the capital of India?",
          "type": "multiple",
        },
        Object {
          "category": "Entertainment: Television",
          "correct_answer": "House Majority Whip",
          "difficulty": "easy",
          "id": 45,
          "incorrect_answers": Array [
            "Attorney General",
            "President",
            "Chief of Staff",
          ],
          "title": "In season one of the Netflix political drama 'House of Cards', what government position does Frank Underwood hold?",
          "type": "multiple",
        },
        Object {
          "category": "History",
          "correct_answer": "Hampton Court",
          "difficulty": "medium",
          "id": 46,
          "incorrect_answers": Array [
            "St James's Palace",
            "Buckingham Palace",
            "Coughton Court",
          ],
          "title": "What was the name of one of the surviving palaces of Henry VIII located near Richmond, London?",
          "type": "multiple",
        },
        Object {
          "category": "Entertainment: Japanese Anime & Manga",
          "correct_answer": "Shiron",
          "difficulty": "hard",
          "id": 47,
          "incorrect_answers": Array [
            "Jack",
            "Axia",
            "Drum",
          ],
          "title": "Which one of these characters is from 'Legendz : Tale of the Dragon Kings'?",
          "type": "multiple",
        },
        Object {
          "category": "Entertainment: Music",
          "correct_answer": "Jones",
          "difficulty": "medium",
          "id": 48,
          "incorrect_answers": Array [
            "Johnson",
            "Edwards",
            "Carter",
          ],
          "title": "What was David Bowie's real surname?",
          "type": "multiple",
        },
        Object {
          "category": "Entertainment: Japanese Anime & Manga",
          "correct_answer": "She crawled into it's mouth as a baby.",
          "difficulty": "medium",
          "id": 49,
          "incorrect_answers": Array [
            "She found it scary.",
            "She was badly injured from it.",
            "It is part Bug.",
          ],
          "title": "In Pok&eacute;mon Chronicles, why was Misty afraid of Gyarados?",
          "type": "multiple",
        },
        Object {
          "category": "Celebrities",
          "correct_answer": "Auction Hunters",
          "difficulty": "medium",
          "id": 50,
          "incorrect_answers": Array [
            "Ramsay's Kitchen Nightmares",
            "Hotel Hell",
            "Hell's Kitchen",
          ],
          "title": "In which of these TV shows did the chef Gordon Ramsay not appear?",
          "type": "multiple",
        },
      ]
    }
  />
</div>
`;
